server/test/services/geo/esri/EsriTestHelper.java

```diff
@@ -4,23 +4,58 @@
  */
 package services.geo.esri;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 import static play.mvc.Results.internalServerError;
 import static play.mvc.Results.ok;
 
+import com.google.common.collect.ImmutableList;
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigFactory;
 import java.io.IOException;
 import java.time.Clock;
 import java.time.ZoneId;
 import java.util.Optional;
+import org.apache.commons.lang3.NotImplementedException;
 import play.libs.ws.WSClient;
 import play.routing.RoutingDsl;
 import play.server.Server;
 import services.geo.AddressLocation;
 import services.settings.SettingsManifest;
 
+/**
```

Comment:
@rockycodes Added some javadoc here to give more context on what the EsriTestHelper class is doing. Also ended up refactoring it because it was already a pain to follow before I made changes and I discovered I had a bug in my new option.

Reply:
This is helpful, thank you!